docs/contribute/index.md

```diff
@@ -17,22 +17,15 @@ to make the process of contributing as smooth as possible.
 3. Create a pull request from a feature branch on your personal fork,
    as explained in the link above, or from a branch of the main repository if you have commit access (you can ask for access on Zulip).
 4. If you've made a lot of changes/additions, try to make many PRs containing small, self-contained pieces. This helps you get feedback as you go along, and it is much easier to review. This is especially important for new contributors.
-5. If you checkout the remote branch `lean-3.5.1` you can fetch the `.olean` binaries using the command
-   ```
-   cache-olean --fetch
-   ```
-   - You can do this by running the following commands (anywhere in the `mathlib` repository).
+5. You can use `leanproject get-cache` to fetch `.olean` binaries.
+  The `.olean` binaries take two hours to generate, so they might not yet be available if you're on the master branch.
+  If you checkout the remote branch `lean-3.6.1` then you can always get olean binaries:
```

Comment:
Unfortunately this is not yet true, this is why I didn't want to merge.

Reply:
The oleans should always be available for `lean-3.6.1` because we first push them to azure before updating the branch.  It's only `master` and feature branches that lag behind.

Reply:
They are now available within ~30 seconds, unless the most recent commit came from a PR from a fork.